packages/ic-vetkd-utils/src/lib.rs

```diff
@@ -331,13 +423,12 @@ impl IBECiphertext {
     ///
     /// For proper operation k_bytes should be the result of calling
     /// TransportSecretKey::decrypt where the same `derived_public_key_bytes`
```

Comment:
`k_bytes` and `TransportSecretKey::decrypt` no longer exist, so the docu should also be adapted.

Reply:
Thanks, updated the doc